docker-compose-debug.yml

```diff
@@ -7,6 +7,7 @@ services:
       - ./db/init:/docker-entrypoint-initdb.d
     ports:
       - 5432
+    tty: true
```

Comment:
Allows seeing colors in the output of docker-compose: https://github.com/docker/compose/issues/2231#issuecomment-165137408

Reply:
that is cool...